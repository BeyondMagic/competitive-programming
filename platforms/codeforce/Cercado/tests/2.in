50 18 15 45
