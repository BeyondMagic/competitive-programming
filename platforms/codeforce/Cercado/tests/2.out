111.8847715
